.travis.yml

```
@@ -86,7 +86,9 @@ script:
   - ./build/push_output.sh
   
   # Run the unit tests
-  # - ./vendor/bin/codecept run unit --steps -f
+  - sudo chmod -R 777 .
+  - echo "\$sugar_config['state_checker']['save_traces'] = false;
```

This doesn't actually do anything. Needs this:
```suggestion
  - echo "\$sugar_config['state_checker']['save_traces'] = false;" >> config_override.php
```
Same for the line below it.